wagtail/admin/templates/wagtailadmin/shared/human_readable_date.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailadmin_tags i18n %}
+
+<button type="button" class="human-readable-date" data-tippy-content="{{ date|date:"DATETIME_FORMAT" }}">
+    <time class="human-readable-date__date" datetime="{{ date|date:"c" }}">
+        {% blocktrans trimmed with time_period=date|timesince_simple %}
+            {{ time_period }}
```

Comment:
This does not need to be in a blocktrans anymore as the value is all that is showing.